### PR TITLE
refactor: address opencensus-nodejs TODOs

### PR DESCRIPTION
**Pull request by eduardoemery**

### CODE REVIEW

File: packages/opencensus-core/src/trace/model/tracer.ts [hunk 1]
```diff
@@ -175,6 +175,17 @@ export class Tracer implements types.Tracer {
     this.eventListenersLocal.push(listner);
   }
 
+  /**
+   * Unregisters an end span event listener.
+   * @param listener The listener to unregister.
+   */
+  unregisterSpanEventListener(listner: types.SpanEventListener) {
```
Comment (kjin) [inline packages/opencensus-core/src/trace/model/tracer.ts]: Nit: `listener` (and in other files as well).

File: packages/opencensus-core/src/trace/types.ts [hunk 2]
```diff
@@ -28,10 +28,13 @@ export interface Tracing {
...
   readonly exporter: exportersTypes.Exporter;
 
+  /** Gets active status  */
+  active: boolean;
+
   /**
-   * Enables the tracing process.
-   * @param userConfig A configuration object.
-   * @returns The tracing object.
+   * Starts the tracing.
```
Comment (kjin) [inline packages/opencensus-core/src/trace/types.ts]: s/`tracing`/`Tracing instance`/g

File: packages/opencensus-core/src/trace/model/tracer.ts (hunk 1 above)
Comment (eduardoemery) [reply packages/opencensus-core/src/trace/model/tracer.ts]: Thanks for noticing.

File: packages/opencensus-core/src/trace/types.ts (hunk 2 above)
Comment (eduardoemery) [reply packages/opencensus-core/src/trace/types.ts]: Ok. Updated here and in the implementation.

**Pull request merged**